src/v/kafka/server/handlers/create_topics.cc

```diff
@@ -30,9 +30,15 @@
 
 namespace kafka {
 
-bool is_not_supported(const ss::sstring& name) {
-    return name == "min.insync.replicas" || name == "flush.messages"
-           || name == "flush.ms";
+bool is_supported(const ss::sstring& name) {
+  return (
+    name == "compression.type" ||
```

Comment:
this could be in a static constexpr array

```
static constexpr std::array<std::string_view, 6> props {{ "cleanup.policy", ... }};
std::string_view nv = name;
return std::any_of(props.begin(), props.end(), [nv](const std::string_view &p){ return nv == p});
```